Sources/Shared/Item.swift

```diff
@@ -66,8 +66,8 @@ public struct Item: Mappable, Indexable {
       Key.Index.string : index,
       Key.Kind.string : kind,
       Key.Size.string : [
-        Key.Width.string : size.width,
-        Key.Height.string : size.height
+        Key.Width.string : Double(size.width),
```

Comment:
Because in init we map it as a `Double`. And seems like `Double` is a default type for floating numbers in json in swift 3.